lib/wordnet/synset.rb

```diff
@@ -100,13 +198,18 @@ def hypernym
       relation(HYPERNYM)[0]
     end
 
+    # Get the parent synset (higher-level category, i.e. fruit -> reproductive_structure).
+    def hypernyms
+      relation(HYPERNYM)
+    end
+
     # Get the child synset(s) (i.e., lower-level categories, i.e. fruit -> edible_fruit)
-    def hyponym
+    def hyponyms
```

Comment:
It's correct; you can have zero or more hyponyms for a synset. I'll keep the old method name around for API consistency, but it's really not great...